scala/ppml/src/main/java/com/intel/analytics/bigdl/ppml/psi/test/BenchmarkClient.java

```diff
@@ -83,13 +83,12 @@ public static void main(String[] args) throws Exception {
 		.usePlaintext()
                 .build();
         try {
-            String[] arg = {"-c", BenchmarkClient.class.getClassLoader()
-                    .getResource("psi/psi-conf.yaml").getPath()};
-            FLClient flClient = new FLClient(arg);
+
+            FLClient flClient = new FLClient();
             flClient.build();
             
             // Get salt from Server
-            salt = flClient.getSalt();
+            salt = flClient.psiStub().getSalt();
```

Comment:
why expose `psiStub` to users here?

Reply:
This is not exposed to user, FLClient is not exposed to user now thus I removed the wrapper in FLClient. User API are `PSI(), LinearRegression`, etc.

So this is simply an internal java test to see if the function works. We could use private package `private [ppml]` if we want to prevent user from using FLClient?